frontend/packages/integration-tests-cypress/views/overview.ts

```diff
@@ -0,0 +1,14 @@
+export const overviewPage = {
+  projectOverviewIsVisible: () => {
```

Comment:
Recommend `projectOverviewShouldBeVisible`  for consistant naming (as the assertion ends in `.should('be.visible')`)
idk, this is kinda how it has evolved.  Personally, I'm ok with 'Is' so feel free to push back  :-)

Reply:
@dtaylor113 done!